src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/CreateAdjudicationRequest.kt

```diff
@@ -19,39 +19,42 @@ data class CreateAdjudicationRequest(
   @Schema(description = "Charges associated with this adjudication")
   val charges: List<ChargeToCreate>,
 
```

Comment:
moved this from Incident to top level which better represents the division between Incident and Adjudication